src/main/resources/templates/teachers_home.html

```diff
@@ -14,78 +17,72 @@
 
     <style>
         html,
-body {
-  height: 100%;
-}
-
-body {
-  display: flex;
-  align-items: center;
-  padding-top: 40px;
-  padding-bottom: 40px;
-  background-color: #f5f5f5;
-}
-
-.form-signin {
-  max-width: 330px;
-  padding: 15px;
-}
-
-.form-signin .form-floating:focus-within {
-  z-index: 2;
-}
-
-.form-signin input[type="email"] {
-  margin-bottom: -1px;
-  border-bottom-right-radius: 0;
-  border-bottom-left-radius: 0;
-}
-
-.form-signin input[type="password"] {
-  margin-bottom: 10px;
-  border-top-left-radius: 0;
-  border-top-right-radius: 0;
-}
+        body {
+            height: 100%;
+        }
+
+        body {
+            display: flex;
+            align-items: center;
+            padding-top: 40px;
+            padding-bottom: 40px;
+            background-color: #f5f5f5;
+        }
+
+        .form-signin {
+            max-width: 330px;
+            padding: 15px;
+        }
+
+        .form-signin .form-floating:focus-within {
+            z-index: 2;
+        }
+
+        .form-signin input[type="email"] {
+            margin-bottom: -1px;
+            border-bottom-right-radius: 0;
+            border-bottom-left-radius: 0;
+        }
+
+        .form-signin input[type="password"] {
+            margin-bottom: 10px;
+            border-top-left-radius: 0;
+            border-top-right-radius: 0;
+        }
 
     </style>
 
 </head>
 <body class="text-center">
 
 
-
 <div class="container-fluid row w-100 m-auto">
 
 
+    <div class="col-4">
+        <h1 class="h3 mb-3">Your last 5 tests</h1>
 
-        <div class="col-4">
-            <h1 class="h3 mb-3">Your last 5 tests</h1>
-
-            <table>
-                <tr th:each="test: ${tests}">
-                    <td th:text="${test.id}" />
-                    <td th:text="${test.name}" />
-                    <td th:text="${test.group.id}" />
-                </tr>
-            </table>
-            <a href="/home/create_test">Add one</a>
-
-        </div>
-
-
-
+        <table>
+            <tr th:each="test: ${tests}">
+                <td th:text="${test.id}"/>
+                <td th:text="${test.name}"/>
+                <td th:text="${test.group.id}"/>
+            </tr>
+        </table>
+        <a href="/home/create_test">Add one</a>
```

Comment:
stick to one case in routes (kebab case)

Reply:
done